test/e2e/storage/testsuites/provisioning.go

```diff
@@ -198,14 +208,31 @@ func testProvisioning(input *provisioningTestInput) {
 		input.pvc.Spec.VolumeMode = &block
 		TestDynamicProvisioning(input.testCase, input.cs, input.pvc, input.sc)
 	})
+
+	It("should provision storage with snapshot data source [Feature:VolumeSnapshotDataSource]", func() {
```

Comment:
Can you also add the feature tag to the alpha suite whitelist [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml#L250) and [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml#L334)

Reply:
Thanks for this information, commit a PR https://github.com/kubernetes/test-infra/pull/11078